File: EJBEngine/ejbModule/com/ts/dt/util/Logger.java
package com.ts.dt.util;

public class Logger {

	public static void log(String message) {
		org.apache.log4j.Logger logger = org.apache.log4j.Logger
				.getLogger("action");
		logger.info(message);
	}

	public static void error(String message) {
		org.apache.log4j.Logger logger = org.apache.log4j.Logger
				.getLogger("action");
		logger.info(message);
	}

}

File: xba_workspace/src/Web/Web/Util/Logger.cs
namespace Web.Util
{
    using log4net;
    using System;

    public class Logger
    {
        private static ILog logger;

        static Logger()
        {
            if (logger == null)
            {
                logger = log4net.LogManager.GetLogger("DEFAULT");
            }
            logger.Info("=======本次日志开始=========");
        }

        public static void Debug(string errMsg)
        {
        }

        public static void Error(string errMsg)
        {
            logger.Error(errMsg);
        }

        public static void Info(string errMsg)
        {
            logger.Info(errMsg);
        }

        public static void Warn(string errMsg)
        {
            logger.Warn(errMsg);
        }

        internal static bool IsDebugEnabled
        {
            get
            {
                return logger.IsDebugEnabled;
            }
        }

        internal static bool IsErrorEnabled
        {
            get
            {
                return logger.IsErrorEnabled;
            }
        }

        internal static bool IsInfoEnabled
        {
            get
            {
                return logger.IsInfoEnabled;
            }
        }

        internal static bool IsWarnEnabled
        {
            get
            {
                return logger.IsWarnEnabled;
            }
        }
    }
}

